ckanext/data_qld/templates/organization/snippets/info.html

```diff
@@ -0,0 +1,16 @@
+{% ckan_extends %}
+
+{% block image %}{% endblock %}
+
+{% block description %}
```

Comment:
comment please, is this from upstream? or reason why we need it?

Reply:
Added comment.